resources/views/_partials/_search.blade.php

```diff
@@ -65,6 +142,4 @@ class="border-0 border-t border-b block pl-10 border-gray-300 md:border md:round
             <a href="https://algolia.com" class="flex justify-end px-4 py-2">
                 <img src="{{ asset('images/algolia.svg') }}" class="h-4 mx-4 my-2" />
             </a>
-        </div>
-    </div>
-</div>
+        </div> --}}
```

Comment:
Why did you comment this out?

Reply:
That's a part from the old code. I forgot to delete it. I'll fix it tomorrow. :)